dll/win32/browseui/lang/fr-FR.rc

```diff
@@ -354,3 +354,9 @@ BEGIN
     IDS_COL_LOCATION "Dans le dossier"
     IDS_COL_RELEVANCE "Pertinence"
 END
+
+STRINGTABLE
+BEGIN
+    IDS_PARSE_ADDR_ERR_TITLE "Explorateur ReactOS"
+    IDS_PARSE_ADDR_ERR_TEXT "L'explorateur ReactOS ne peut trouver '%1'. Veuillez vérifier l'orthographe et réessayer."
```

Comment:
Current windows text is "Windows ne parvient pas à trouver '%1'.  Vérifiez l'orthographe et réessayer."

Reply:
Either one works for me - the current commit uses a suggestion from HBelusca above.  I'd prefer to not get caught up in translation details and have future PRs submitted from translators to localize the resources.  Would that work?

Reply:
For the title : OK for your proposal
For the text : @HBelusca : would you agree on my suggestion ?

Reply:
@giawa that is perfectly fine, feel free to not apply translations at all unless you want to.

Reply:
Agreed on my side.

Reply:
OK so @giawa, please update your proposal for  IDS_PARSE_ADDR_ERR_TEXT and let's go :)

Reply:
Sorry Kyle, got merged before I could push that change.  We'll have to do it via another PR.

Reply:
OK, I take care of this.